File: java/Problem1048.java
import java.util.Scanner;

public class Problem1048 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        double salario = sc.nextDouble();
        double percentual;
        double newSal;

        if (salario >= 0 && salario <= 400) {
            percentual = 0.15;
        } else if (salario > 400 && salario <= 800){
            percentual = 0.12;
        } else if (salario > 800 && salario <= 1200){
            percentual = 0.1;
        } else if (salario > 1200 && salario <= 2000){
            percentual = 0.07;
        } else {
            percentual = 0.04;
        }


        newSal = salario + (salario * percentual);

        System.out.printf("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: %.0f %%\n", newSal, (newSal - salario), (percentual * 100));
        
        sc.close();
    }
}

File: python/problem1048.py
entrada = input();
salario = float(entrada);

if salario >= 0 and salario <= 400:
	porcentagem = 0.15;
elif salario > 400 and salario <= 800:
	porcentagem = 0.12;
elif salario > 800 and salario <= 1200:
	porcentagem = 0.10;
elif salario > 1200 and salario <= 2000:
	porcentagem = 0.07;
elif salario > 2000:
	porcentagem = 0.04;
else:
	print("Pode não man");

ajuste = salario * porcentagem;
salario += ajuste;
porcentagem *= 100;
percent = int(porcentagem);
percent = int(porcentagem);
print("Novo salario: %.2f\nReajuste ganho: %.2f\nEm percentual: %d" % (salario, ajuste, percent) + ' %');
